tests/python/unittest/test_ndarray.py

```diff
@@ -593,6 +593,10 @@ def gt_topk(dat, axis, ret_typ, k, is_ascend):
     nd_ret_argsort = mx.nd.argsort(a_nd, axis=None, is_ascend=False).asnumpy()
     gt = gt_topk(a_npy, axis=None, ret_typ="indices", k=5*5*5*5, is_ascend=False)
     assert_almost_equal(nd_ret_argsort, gt)
+    
+    # test topk with a big shape
+    a = mx.nd.arange(0, 54686454, step=1, repeat=1)
```

Comment:
Why would int->int64 have any effect on this? 54686454 is less than max int

Reply:
After discussing with @sxjscience , we found that https://github.com/apache/incubator-mxnet/issues/8303 is because of the fix for range in mshadow https://github.com/dmlc/mshadow/commit/eced9571c601260e3a2deb0e8e9a41e8c0a9f0f1 is not included in mxnet master (range still use float).

So just replacing `range` with kernel launch can fix this issue. I change `int64` back to `int`. Or is `int64` better than `int` here?

Reply:
OK lets change range to kernel launch since we are deprecating mshadow. int is fine for now. Leave a note and we'll support int64 when we do the int64 index support in general.

Reply:
Is there any other way to test this rather than this big ndarray?

Reply:
Is not clear what was the problem and how the fix works.

Reply:
I think the problem is with the range being stored in a float, values bigger than the mantissa 2^24 will get truncated.